src/components/Dropdown/Dropdown.scss

```diff
@@ -32,14 +35,23 @@
         display: flex;
         flex-direction: column;
         align-items: flex-end;
+
+        border: 1px solid #e0e0e0;
+        background-color: white;
       }
   
       &.hide {
         display: none;
       }
 
       .dropdown-item {
-        margin: 5px 0;
```

Comment:
removed margin and added it to padding so that the "hoverable area" looks bigger